aries-site/src/pages/components/maskedinput.js

```diff
@@ -22,11 +34,178 @@ const MaskedInput = () => {
           <Example
             code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/MaskedInputExample/MaskedInputExample.js"
             docs="https://v2.grommet.io/maskedinput?theme=hpe#props"
+            figma="https://www.figma.com/file/fO285pXUqN9pVZ0xpRoKjL/HPE-Masked-Input-Component?node-id=1%3A89"
           >
-            <MaskedInputExample />
+            <MaskedPhoneExample />
           </Example>
         </Subsection>
       </ContentSection>
+      <ContentSection>
+        <Subsection name="Guidance">
+          <SubsectionText>
+            MaskedInput place syntax restrictions on the text that can be
+            entered in an input field. It is similar to{' '}
+            <Anchor label="TextInput" href={nameToPath('TextInput')} />, in that
+            is allows the user to input shorter forms of data and content.
+            However, it is used to provide extra guidance and enforcement of
+            input formats.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Usage" level={3} gap="small">
```

Comment:
Wondering if we can talk about / give example for how MaskedInput be a replacement / reduce multiple input fields into one and simplify common interfaces? For example, consider an e-commerce site with a product results list. What if instead of having all the filtering actions in the sidebar with menus and checkboxes, the entire sidebar was replaced by a single mask (e.g. running mens size 10.5 blue)? The Storage Size example would be similar.

Not sure those are the best cases to showcase replacement, but I think it is interesting concept to explore / seed with the design system audience.

Reply:
I will use the Storage Size example of this for now, but I'll add some copy the explains the benefit and links to it.